test/utils/helpers.go

```diff
@@ -188,6 +188,20 @@ func GetServiceClusterIPPort(k8sClient client.Client, name, namespace string) (s
 	return service.Spec.ClusterIP, service.Spec.Ports[0].Port, nil
 }
 
+// AddSecret adds a Secret to namespace from the cloud account parameters.
+func AddSecret(kubeCtl *KubeCtl, params k8stemplates.CloudAccountParameters) error {
```

Comment:
We dont have AddSecret routine? Could you add other test. Also, how is this removed?

Reply:
Update function to handle Add/Remove for Secret and Account. Also update removing stale Secret created by integration tests.